edx_exams/apps/api/v1/views.py

```diff
@@ -5,17 +5,22 @@
 import uuid
 
 from django.core.exceptions import ObjectDoesNotExist
+from django.utils import timezone
 from edx_api_doc_tools import path_parameter, schema
 from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
 from rest_framework import status
 from rest_framework.generics import ListAPIView
 from rest_framework.response import Response
 from rest_framework.views import APIView
+from token_utils.sign import create_jwt
```

Comment:
just noticed that this isn't using the public API for token utils. We should be using the api here https://github.com/edx/token-utils/blob/main/token_utils/api.py#L8